From b47fc5dcf3c51b66005c9413f6543598de4d41bc Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Wed, 12 Nov 2014 19:08:04 -0500
Subject: [ethernet] cxgb4: Add Devicde ID for two more adapter

Message-id: <1415819300-32670-4-git-send-email-svemuri@redhat.com>
Patchwork-id: 99860
O-Subject: [RHEL7.1 PATCH BZ 1163467 03/19] cxgb4/cxgb4vf: Add Devicde ID for two more adapter
Bugzilla: 1163467
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: David S. Miller <davem@redhat.com>

Signed-off-by: Hariprasad Shenai <hariprasad@chelsio.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 91c04a9eb3a9b7e340a59446989263bfe31aee62)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
index f034ee8..ee2e32a 100644
--- a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
@@ -283,6 +283,8 @@ static DEFINE_PCI_DEVICE_TABLE(cxgb4_pci_tbl) = {
  CH_DEVICE(0x5084, 4),
  CH_DEVICE(0x5085, 4),
  CH_DEVICE(0x5086, 4),
+ CH_DEVICE(0x5087, 4),
+ CH_DEVICE(0x5088, 4),
  CH_DEVICE(0x5401, 4),
  CH_DEVICE(0x5402, 4),
  CH_DEVICE(0x5403, 4),
@@ -311,6 +313,8 @@ static DEFINE_PCI_DEVICE_TABLE(cxgb4_pci_tbl) = {
  CH_DEVICE(0x5484, 4),
  CH_DEVICE(0x5485, 4),
  CH_DEVICE(0x5486, 4),
+ CH_DEVICE(0x5487, 4),
+ CH_DEVICE(0x5488, 4),
  { 0, }
 };
 
diff --git a/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c b/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
index 305fd9b..0e3d3fd 100644
--- a/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
@@ -2961,6 +2961,8 @@ static const struct pci_device_id cxgb4vf_pci_tbl[] = {
  CH_DEVICE(0x5884),
  CH_DEVICE(0x5885),
  CH_DEVICE(0x5886),
+ CH_DEVICE(0x5887),
+ CH_DEVICE(0x5888),
  { 0, }
 };
 
-- 
1.7.1